src/math/inRange.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * Checks if value is inside the range.
  */
-function inRange(val: number, min: number, max: number, threshold = 0) {
+function inRange(val: number, min: number, max: number, threshold?: number = 0) {
```

Comment:
While this is totally correct, one doesn't actually have to specify the type when a default paramter is given, as the compiler deduces `number` from it, as well as the optionality, since it has a default option.